manifests/init.pp

```diff
@@ -30,7 +33,8 @@
   $target_dir   = '/usr/local/bin',
   $with_npm     = true,
   $make_install = true,
-  $node_path    = '/usr/local/node/node-default/lib/node_modules'
+  $node_path    = '/usr/local/node/node-default/lib/node_modules',
+  $python_package = 'python',
```

Comment:
the diff is hidden, so it is outdated. It seems as you reloaded before github detected the commited changes